lib/cocoapods/generator/header.rb

```diff
@@ -82,7 +82,7 @@ def save_as(path)
       # @return [String]
       #
       def generate_platform_import_header
-        "#import #{platform == :ios ? '<UIKit/UIKit.h>' : '<Cocoa/Cocoa.h>'}\n"
+        "#import <Foundation/Foundation.h>\n"
```

Comment:
Possibly - might be a better idea to only use `Foundation` for `watchos` targets.